src/app/design/styles/color/index.html

```diff
@@ -356,10 +356,10 @@
 		
 	<div>
   <stache-page-anchor>
-    Containers
+    Categorization
   </stache-page-anchor>
     <p>
-    Use container colors to categorize blocks of content that mix together. For example, use container colors for different types of events in calendars. Container colors can be chosen by variable name or applied in a sequence.
+    Use categorization colors to categorize blocks of content that mix together. For example, use categorization colors for different types of events in calendars. These colors can be chosen by variable name or applied in a sequence.
```

Comment:
I'd change "that mix together" to "within the same container"

And I'd tweak the last sentence to use active voice: "You can select colors for content categories by variable name or apply them in a sequence."